index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
```

Comment:
- [ ] As you may notice here in your head, there's nowhere your HTML markup is linked with the stylesheet CSS. Please make sure you fix that so that your styles may be applied.
```html
You may consider saying:
<link rel="stylesheet" type="text/css" href="styles.css" />
```

Reply:
I'll include the link in the HTML, I didn't notice that @christianbiring1.